Src/Models/CBR.cpp

```diff
@@ -87,6 +114,9 @@ CBR::CBR(GraphicsResources* gr, const PGE::String& filename) {
             reader.skip(2 * 3 * 4 + 5 * 16);
             int vertexCount = reader.readInt();
             int vertexOffset = vertices[textureID].size();
+
+            std::vector<int> indices;
```

Comment:
What is the purpose of this? Primitives are our indices.

Reply:
This is a mash-up of both formats to get something functional.
There is a certain satisfaction to factually knowing it works in concept.
Further iteration required.

Reply:
I'm not sure what you mean? This variable is just declared and never used.